CMakeLists.txt

```diff
@@ -22,13 +21,9 @@ if ("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fcolor-diagnostics")
 endif()
 
-# Because we use ninja, we have to explicitly turn on color output for the compiler
-if ("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang")
```

Comment:
That block was in there twice from a previous merge. It's still there - just once now.